.github/workflows/deploy.yaml

```diff
@@ -7,7 +7,9 @@ on:
 jobs:
   deployment:
     runs-on: ubuntu-latest
-    environment: staging_environment
+    environment: 
+      name: staging_environment
```

Comment:
```suggestion
      name: staging
```
?

Reply:
Yea i'll do that some time but it requires me to recreate the environment in the repo settings and copy all the secrets again